### DISCUSSION TITLE
Pterodactyl and Pelican

### DISCUSSION THREAD

**Discussion by louanfontenele** [coollabsio/coolify#4215] [Service Template Requests]
It would be nice to have Pterodactyl and Pelican both are game managers, and can be used to host games and other things too, it's a great addition.

https://pterodactyl.io/
https://pelican.dev/

**Comment by maxbailey** [comment]
Bump. The Pterodactyl installation script conflicts with Coolify when trying to run, it would be great to just be able to spin up a service through Coolify instead.

**Comment by TesterHere06** [comment]
Bump. Pterodactyl/Pelican are perfect for game hosting, as well as a couple other needs and it would be amazing to have them on here

**Comment by aidendotgg** [comment]
Would be great to have

**Comment by straatrakker** [comment]
Would be great

**Comment by cowboycodr** [comment]
+1 Seems like this would be simple enough. Just need to configure [peterodactyl's docker compose file](https://github.com/pterodactyl/panel/blob/1.0-develop/docker-compose.example.yml) and plop it in [this folder](https://github.com/coollabsio/coolify/tree/592dd76940f0bbaf233eb2b5fea713bf9bc5d429/templates/compose).

**Comment by Telokis** [comment]
See https://github.com/coollabsio/coolify/pull/3052

**Reply by hades200082** [reply]
I see the pull was merged but don't see it in Coolify. Have I missed something?

**Reply by laGameTV** [reply]
> Good news, I got a semi-automated install working, but it is not fully working yet, so I leave it disabled for now. Of course if someone wants to pick it up, just open a PR. Sorry for closing it first, merging makes more sense, especially since it almost works.

_Originally posted by @peaklabs-dev in https://github.com/coollabsio/coolify/issues/3052#issuecomment-2512319823_

**Comment by laGameTV** [comment]
bump for pelican!

**Comment by LilNesquuik** [comment]
bump for Pterodactyl !

**Comment by ambroseblot** [comment]
bump for both!

**Comment by rasouza** [comment]
bump for both :D

**Comment by Default-01** [comment]
Bump aswell for both. Or some kind of support for hosting gameservers with an extensive list of game servers. so you don't have a pterodactyl panel inside coolify which is not really needed IMO.

**Comment by jlengelbrecht** [comment]
Bump. Please add pterdactyl

**Comment by Trebossalol** [comment]
➕bump

**Comment by mishl-dev** [comment]
bump

**Comment by JesperKuipers** [comment]
bump for pelican [it has docker compose](https://pelican.dev/docs/panel/advanced/docker/)

**Comment by alepouna** [comment]
For anyone checking in today, Pterodactyl with just the panel, wings, or both, is available since Aug 11, 2024 and Aug 11, 2025 (a year later lol) it got split. Might be worth to mark this as resolved @peaklabs-dev @Cinzya (and split the Pelican suggestion)

**Comment by yepMad** [comment]
Ptero is broken. The MySQL service does not start, only MariaDB. The default Ptero configuration expects a mysql.

**Reply by PandomaOes** [reply]
Im not sure if this helps, i tried installing today and cant login to the panel.

What i did was go into the panel terminal and did these 2:
apk add --no-cache mariadb-client
php artisan migrate --seed --force

then restart service. (because troubleshooting lead me to the reason of a missing mysql client)
